chart/templates/mayastor/agents/core/agent-core-deployment.yaml

```diff
@@ -43,7 +43,7 @@ spec:
             requests:
               cpu: {{ .Values.agents.core.resources.requests.cpu | quote }}
               memory: {{ .Values.agents.core.resources.requests.memory | quote }}
-          image: "{{ .Values.image.registry }}/{{ .Values.image.repo }}/{{ .Chart.Name }}-agent-core:{{ default .Values.image.tag .Values.image.repoTags.controlPlane }}"
+          image: "{{ .Values.image.registry }}/{{ .Values.image.repo }}/{{ .Values.image.chart }}-agent-core:{{ default .Values.image.tag .Values.image.repoTags.controlPlane }}"
```

Comment:
This looks like it would break things. There is no `image.chart` in the values.yaml file. Should it be `image.prefix` instead.

Reply:
will fix this, I had this in pro chart but missed it here.

Reply:
this is fixed now.